### DISCUSSION TITLE
Enhancement Request: Temperature Sensor Naming (ASUS Motherboards)

### DISCUSSION THREAD

**Discussion by TheAndyMac** [Rem0o/FanControl.Releases#1930] [Ideas]
It would be really helpful if the Temperature Sensors were identified as to their function and not just the internal name. On ASUS motherboards there are often 5 or more sensors (often #1, #2, #4, #5, #6) and it is quite difficult to determine which ones should be used for fan control. Even if the key ones were identified specifically, e.g. Motherboard, VRM, PCH, etc. that would be helpful.

Maybe extend this by offering some pre-determined custom sensor mixes for specific ranges of motherboard through the Assisted Setup (and Assisted Setup Steps) to help newbies.

**Comment by Rem0o** [comment]
Names come from : https://github.com/LibreHardwareMonitor/LibreHardwareMonitor

Some boards have their sensors identified with meaningful names, but obviously, not every single board on the market is identified there.
